Lib/test/test_importlib/test_lazy.py

```diff
@@ -140,6 +146,36 @@ def test_module_already_in_sys(self):
             # Force the load; just care that no exception is raised.
             module.__name__
 
+    def test_module_load_race(self):
```

Comment:
This needs the `test.support.threading_helper.requires_working_threading()` decorator to prevent running on platforms lacking threads.

```suggestion
    @threading_helper.requires_working_threading()
    def test_module_load_race(self):
```